metacat-connector-hive/src/main/java/com/netflix/metacat/connector/hive/HiveConnectorFactory.java

```diff
@@ -56,7 +56,7 @@ public class HiveConnectorFactory extends SpringConnectorFactory {
             connectorContext.getConfiguration()
                 .getOrDefault(HiveConfigConstants.USE_EMBEDDED_METASTORE, "false")
         );
-        final boolean useFastHiveService = useLocalMetastore && Boolean.parseBoolean(
+        final boolean useFastHiveService = Boolean.parseBoolean(
```

Comment:
Do we need this anymore? I feel we should remove the embedded HMS entirely given it won't work anymore on SBN3.